lib/render/editor/editor.html.erb

```diff
@@ -376,8 +376,17 @@
         return mumuki.exercise.layout === 'input_kindergarten';
       },
 
+      _isGame() {
+        if (mumuki.exercise.settings) {
+          return mumuki.exercise.settings.game_framwork;
```

Comment:
Talking about typos :rofl:  framework* 
I'll correct it and merge it!

Reply:
Thanks!